Code/GraphMol/Descriptors/Wrap/rdMolDescriptors.cpp

```diff
@@ -1396,6 +1408,13 @@ BOOST_PYTHON_MODULE(rdMolDescriptors) {
               (python::arg("mol")), docString.c_str(),
               python::return_value_policy<python::manage_new_object>());
 
+  python::scope().attr("_GetAtomFeatures_version") = RDKit::Descriptors::AtomFeatVersion;
+  docString = "Returns the Atom Features vector";
+  python::def(
+      "GetAtomFeatures", GetAtomFeatures,
+      (python::arg("mol"), python::arg("atomid") = 0, python::arg("addchirald") = false),
```

Comment:
I don't think the atomid should have a default value (i.e. the user should always provide it).
Also the third argument should probably be named `addchiral` not `addchirald`